proof/crefine/ARM/IsolatedThreadAction.thy

```diff
@@ -233,6 +247,18 @@ lemma zipWithM_setRegister_simple:
                             simpler_modify_def fun_upd_def[symmetric])
   done
 
+(* these variant used in fastpath rewrite proof *)
+lemma setRegister_simple_modify_registers:
+  "setRegister r v = (\<lambda>con. ({((), modify_registers (\<lambda>f. f(r := v)) con)}, False))"
+  by (simp add: modify_registers_def setRegister_simple)
+
+(* these variant used in fastpath rewrite proof *)
```

Comment:
```suggestion
(* this variant used in the fastpath rewrite proof *)
```

Reply:
I was trying to say this one and `zipWithM_setRegister_simple_modify_registers` are used for the fastpath rewrite proof, so at least I tried to write "these variants" but clearly only made it halfway. Do you think the same comment above both then, or could people infer that the `_modify_registers` variants are related?